server/templates/pages/src/index.html

```diff
@@ -1,7 +1,5 @@
 <!doctype html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="lt-ie10 lt-ie9"> <![endif]-->
+<!--[if lt IE 9]>      <html class="no-js"> <![endif]-->
 <!--[if IE 9]>         <html class="lt-ie10"> <![endif]-->
```

Comment:
The browse happy text is still visible in IE8. Let's update for consistency. Change to `if lt 10`.